test/e2e/specs/onboarding/heroflow__write.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	StartSiteFlow,
+	RestAPIClient,
+	SecretsManager,
+	NewSiteResponse,
+	TestAccount,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+import { apiDeleteSite } from '../shared';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Onboarding heroflow: write' ), function () {
```

Comment:
**suggestion::nonblocking**: could the suite name be renamed to `FTME: Write` to be in sync with [this spec file](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/onboarding/ftme__sell.ts)?

In similar vein..

**suggestion::nonblocking**: could the file name be renamed to `ftme__write`, for similar reasons?